Publication 20250112 (#106)
yayoi-chi/yaya-world: content/en/news/2025/stream-recap-12-jan.org

#+TITLE: Stream Recap: 12 Jan
#+DATE: 2025-01-12T09:58:38-08:00
#+DRAFT: false
#+DESCRIPTION:
#+TAGS[]: stream recap news
#+KEYWORDS[]:
#+SLUG:

> #+SUMMARY: I much more typical week after taking so much time off for holidays! We finished another page, started another illustration, went back to Hollownest, and more! I enjoyed getting to spend more time with the kittens! Hope you enjoyed it too~ 💜

Reviewer: I think you meant A not I?

```suggestion
#+SUMMARY: A much more typical week after taking so much time off for holidays! We finished another page, started another illustration, went back to Hollownest, and more! I enjoyed getting to spend more time with the kittens! Hope you enjoyed it too~ 💜
```

* Art Streams

> Did two art streams this week. I mentioned this before, but I really do like doing art streams and will probably continue to do more. For this week, I completed page three of my (still unnamed) comic. I wanted to take a bit of a break from it though, so I started another illustration. Honestly, I was hoping that illustration would be done in one stream, but I guess I'm not that good yet. Maybe one or two more stream. I want to use it on a schedule next week (please don't look at the image for this coming week's schedule). Maybe I'll start getting faster as I continue to practice!

Reviewer: Maybe consider different word from 'another'. I think "a new" flows well!
```suggestion
Did two art streams this week. I mentioned this before, but I really do like doing art streams and will probably continue to do more. For this week, I completed page three of my (still unnamed) comic. I wanted to take a bit of a break from it though, so I started a new illustration. Honestly, I was hoping that illustration would be done in one stream, but I guess I'm not that good yet. Maybe one or two more stream. I want to use it on a schedule next week (please don't look at the image for this coming week's schedule). Maybe I'll start getting faster as I continue to practice!
```

Author: Thank you, this sounds good!

#+begin_export html
<iframe width="560" height="315" src="https://www.youtube.com/embed/XpQyPaAYgNs?si=S0lUtlgmbDHldUU8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
#+end_export
* Game Streams

> Two art stream do many one less game stream. I promise I won't quit doing them though. I like sharing my art with you and I get more chances to chat when I do, but I also like playing games for you. I'll keep doing both as long as I can! This week we went back to Hollow Knight. I need a bit more direction on where to go next though (any hints? 😉). I also fed my Genshin addiction by continuing the dendro archon quest...for a bit...until I got sucked into another side quest 😅

Reviewer: Honestly, not sure what you meant to say. I tried making sense of it though, hopefully I got it right.
```suggestion
Two art stream does mean one less game stream. I promise I won't quit doing them though. I like sharing my art with you, and I get more chances to chat when I do, but I also like playing games for you. I'll keep doing both as long as I can! This week we went back to Hollow Knight. I need a bit more direction on where to go next though (any hints? 😉). I also fed my Genshin addiction by continuing the dendro archon quest...for a bit...until I got sucked into another side quest 😅
```

** Hollow Knight
I'm so lost! I mean I've seemingly explored most of Hollownest, but there is still at least one spot I can't get to! I spent much of this stream trying to find a way over there, but never did make it. I found a few other small things I missed, but nothing helpful. I think I'll need to turn to you, my kittens, or the interwebs for direction!
#+begin_export html
<iframe width="560" height="315" src="https://www.youtube.com/embed/pY558p3pVek?si=tnesU6OhDf96G6R-" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
#+end_export
** Xenosaga Episode 1

> Our rematch with Mintia got a little hairy when I forgot that she can take someone down to 1 HP and use that the heal herself that amount. We managed to get passed that with boosts though. After we I got the Dark Septer tech attack and the new rod, we moved on to Proto Merkabah. Wasn't there very long though! We'll get through more of it next time!

Reviewer: did you are we get the scepter?
```suggestion
Our rematch with Mintia got a little hairy when I forgot that she can take someone down to 1 HP and use that the heal herself that amount. We managed to get passed that with boosts though. After we got the Dark Scepter tech attack and the new rod, we moved on to Proto Merkabah. Wasn't there very long though! We'll get through more of it next time!
```
